Simulator/simulation/simulator.js

```diff
@@ -72,15 +76,16 @@ Simulator = {
 	runSimulation: async function() {
 		await this.initHouses();
 		this.timestamp = Date.now();
+		Noise.generateWindMap(Math.random());
 		setInterval(this.simulationLoop, this.pollTime, this);
 	},
 	calcPrice: function() {
 		const cd = 10 ** 6; // Coeff for demand variable
 		const cs = 1; // Coeff for supply variable
 		this.price = cd / this.power + cs * this.powerLoss;
 
-		console.log('Power: ' + this.power.toFixed(2) + ' -- Price: ' +
-						this.price.toFixed(2)+' -- Loss: '+this.powerLoss.toFixed(2));
+		// console.log('Power: ' + this.power.toFixed(2) + ' -- Price: ' +
+		// this.price.toFixed(2)+' -- Loss: '+this.powerLoss.toFixed(2));
```

Comment:
Stop it. Get some help

Reply:
its hard to see if a simulation works if you have no output